scripts/src/main/resources/scripts/command/aws

```diff
@@ -29,43 +43,46 @@ function doPackageInstall()
   then
     doEcho "Installing AWS for Windows..."
     windows_path_to_package=$(cygpath -w "${path_to_package}")
-    powershell.exe -Command "Start-Process msiexec.exe -verb runas -Wait -ArgumentList '/I ${windows_path_to_package}\aws-${current_version}-AWSCLIV2.msi /quiet'"
+    powershell.exe -Command "Start-Process msiexec.exe -verb runas -Wait -ArgumentList '/I ${windows_path_to_package}\aws-${AWS_VERSION}-windows.msi /quiet'"
   elif doIsMacOs
   then
     doFail "Sorry, AWS CLI installation support is not yet implemented for your OS. Please install manually or help devonfw-ide to support it for your OS by contributing a pull-request."
   else
     doEcho "Installating AWS for Linux..."
     doRunCommand "sudo ${path_to_package}/install"
   fi
-  doRunCommand "rm -rf ${path_to_package}"
+  doRunCommand "rm -rf ${path_to_package}"  
 }
 
 function doSetup() {
-
-  if [ ! -d "${AWS_HOME}" ] || [ "${1}" != "silent" ]
+  if ! doIsAwsInstalled
   then
-    # Get leatest release
-    if [ -z "${AWS_VERSION}" ]
+    if [ ! -d "${AWS_HOME}" ] || [ "${1}" != "silent" ]
     then
-      doEcho "Getting latest release..."
-      AWS_VERSION=$(curl "https://github.com/aws/aws-cli/tags" | awk -F'tags/' '/archive/ { print $2}' | sort -r | head -1 | awk -F'.zip' '{print $1}')
-    fi
+      # Get leatest release
+      if [ -z "${AWS_VERSION}" ]
+      then
+        doEcho "Getting latest release..."
+        AWS_VERSION=$(curl "https://github.com/aws/aws-cli/tags" | awk -F'tags/' '/archive/ { print $2}' | sort -r | head -1 | awk -F'.zip' '{print $1}')
+      fi
 
-    doInstall "-" "${AWS_HOME}" "aws" "${AWS_VERSION}" 
-    if [ "${?}" = 0 ]
-    then
-      doPackageInstall "${AWS_HOME}"
+      doInstall "-" "${DEVON_IDE_HOME}/updates/install/aws" "aws" "${AWS_VERSION}" 
+      if [ "${?}" = 0 ]
+      then
+        doPackageInstall "${DEVON_IDE_HOME}/updates/install/aws"
+      fi
+
+      if [ "${1}" != "silent" ] && ! doIsQuiet
+      then
+        doEcho "logon again to test aws with \"aws --version\""
+      fi 
     fi
+  else
+    echo "aws is already installed"
+    doRunCommand "aws --version"
```

Comment:
`silent` means that if the tool is already is installed, then nothing should be logged and almost nothing should happen (that takes time or requires network access). So this `else` block should only happen if not silent.
The regular thing to do that in all other commandlets is that separate `if` that you removed here:
```
if [ "${1}" != "silent" ] && ! doIsQuiet
```
You can simply remove the message "aws is already installed".

Reply:
It misunderstood the word "silent". Now I corrected it